site/pages/docs/ref/wb-jsonmanager/wb-jsonmanager-en.hbs

```diff
@@ -232,402 +243,405 @@
-</tr>
-<tr>
-<td><code>value</code></td>
-<td><code>{ "filter": [ { "value": "A value" } ] &hellip;}</code> or <code>{ "filternot": [ { "value": "A value" } ] &hellip;}</code> </td>
-<td>Any value that could be compared with the information retreived form the path. An exact match is performed.</td>
-</tr>
-<tr class="text-muted">
-<td><code>optional</code></td>
-<td><code>{ "filter": [ { "optional": true } ] &hellip;}'</code> or <code>{ "filternot": [ { "optional": true } ] &hellip;}'</code></td>
-<td>True or false. If omited it will be false by default.</td>
-</tr>
-</tbody>
-</table>
-<h3 id="extractobj">Extractor (selector object)</h3>
-<table class="table">
-<thead>
-<tr>
-<th>Option</th>
-<th>How to configure</th>
-<th>Values and clarification</th>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td><code>selector</code></td>
-<td><code>{ "extractor": [ { "selector": "h2" } ] &hellip;}</code> or <code>{ "extractor": [ { "selector": "dl[data-dl]" } ] &hellip;}</code></td>
-<td>A valid css selector for any html element from the page </td>
-</tr>
-<tr>
-<td><code>path</code></td>
-<td><code>{ "extractor": [ { "path": "firstH2" } ] &hellip;}</code> or <code>{ "extractor": [ { "path": "myGroup" } ] &hellip;}</code></td>
-<td>The value of this property will be the pointer in the JSON object dataset that will be created</td>
-</tr>
-<tr>
-<td><code>selectAll</code></td>
-<td><code>{ "extractor": [ { "selectAll": true } ] &hellip;}</code> </td>
-<td>If true, it will search in the page for  all the html elements indicated in the <code>selector</code> property</td>
-</tr>
-<tr>
-<td><code>attr</code></td>
-<td><code>{ "extractor": [ { "attr":"data-td" } ] &hellip;}</code> </td>
-<td>If present, it will collect the value from the specified attribute for the mentioned selector instead of the text value of the <code>selector</code> property</td>
-</tr>
-<tr>
-<td><code>interface</code></td>
-<td><code>{ "extractor": [ { "interface":"referer" } ] &hellip;}</code> or <code>{ "extractor": [ { "interface":"locationHref" } ] &hellip;}</code> </td>
-<td>If present, it will collect the value from predefined properties as follows <span class="text-muted">(This list is not exhaustive):</span>
-	<ul><li><code>referer</code> is assigned for the <code>document.referrer</code> value</li><li><code>locationHref</code> is assigned for <code>location.href</code> value</li></ul></td>
-</tr>
-<tr>
-<td><code>extractor</code></td>
-<td><code>{ "extractor": [ { "extractor":[ {
-									"selector": "dt",
-									"path": "title",
-									"attr":"data-td"
-								},
-								{
-									"selector": "dd",
-									"path": "desc"
-								}] } ] &hellip;}</code> </td>
-<td>Used for combined selectors e.g. description lists. Same configuration as for outer <code>extractor</code> option, it is an array of objects containing html selectors with their specific paths.
-
-</tr>
-</tbody>
-</table>
+	<h2>Configuration options</h2>
```

Comment:
Need to add a blurb for the new option "concat"

Reply:
Following our discussion, the "concat" option has been removed. Arrays will be concatenated by default and objects extended.